airbyte-integrations/connectors/source-looker/unit_tests/unit_test.py

```diff
@@ -2,7 +2,11 @@
 # Copyright (c) 2022 Airbyte, Inc., all rights reserved.
 #
 
+from source_looker.streams import LookerStream
 
-def test_example_method() -> None:
-    assert True
-    return
+
+def test_format_null_in_schema():
```

Comment:
can you add a test case that uses the recursion?

Reply:
updated